unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsHUD/QuestTaskUIEntry.cs

```diff
@@ -0,0 +1,41 @@
+using DCL.Helpers;
+using TMPro;
+using UnityEngine;
+
+public class QuestTaskUIEntry : MonoBehaviour
```

Comment:
Maybe we can put all the classes under the DCL namespace?

Reply:
Sure thing.

What do you think about `DCL.Huds`?

Reply:
Yeah, I believe it looks good